repose-aggregator/components/services/opentracing-service/opentracing-service-impl/src/main/scala/org/openrepose/core/services/opentracing/TracerExtractor.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.core.services.opentracing
+
+import java.util
+import java.util.AbstractMap.SimpleImmutableEntry
+
+import com.typesafe.scalalogging.slf4j.LazyLogging
+import io.opentracing.propagation.TextMap
+import org.openrepose.commons.utils.servlet.http.HttpServletRequestWrapper
+
+import scala.collection.mutable.ListBuffer
+
+class TracerExtractor() extends TextMap with LazyLogging {
```

Comment:
Why have a default constructor that you never use?

Just make this class signature:
```
class TracerExtractor(httpServletRequestWrapper: HttpServletRequestWrapper) extends TextMap with LazyLogging {
```